README.md

```diff
@@ -5,6 +5,11 @@ This repository is a template where to start to create [MapStore Extensions](htt
 It is basically a customized MapStore project that allows to run, test and build a sample extension.
 You can copy this repository and modify the sample extension to develop your own one.
 
+From 22-08-2022 we started following the release branching procedure we have on main MapStore project. This means two things:
+
+- **master** branch here will follow and submodule revision will be aligned to master branch [here](https://github.com/geosolutions-it/MapStore2)
+- **stable branch** will do the same, will follow and submodule revision will be aligned to latest stable branch available, which currently is [2022.02.xx](https://github.com/geosolutions-it/MapStore2/tree/2022.02.xx)
```

Comment:
You may need to udpdate the readme on every single update.
Given that this repo will be replaced by mapstore-project soon, I should not indicate the name of the latest to avoid to update it everytime, or you should add a step to the cecklist to manually update it.

Reply:
> You may need to udpdate the readme on every single update.

only if it there is some gha that does that otherwise its not susstainable

> Given that this repo will be replaced by mapstore-project soon, I should not indicate the name of the latest to avoid to update it everytime, or you should add a step to the cecklist to manually update it.

i can remove the reference